_posts/en/newsletters/2021-08-11-newsletter.md

```diff
@@ -0,0 +1,168 @@
+
+{% include specials/taproot/en/07-nonces.md %}
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.1rc2][C-Lightning 0.10.1] is a release candidate for
+  an upgrade that contains a number of new features, several bug fixes,
+  and a few updates to developing protocols (including [dual
+  funding][topic dual funding] and [offers][topic offers]).
+
+- [Bitcoin Core 22.0rc2][bitcoin core 22.0] is a release candidate
+  for the next major version of this full node implementation and its
+  associated wallet and other software. Major changes in this new
+  version include support for [I2P][topic anonymity networks] connections,
+  removal of support for [version 2 Tor][topic anonymity networks] connections,
+  and enhanced support for hardware wallets.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core #21528][] aims to improve the propagation of IP addresses
+  in the p2p network. Exposure to a diverse set of addresses is important
+  for nodes to be protected against network partitions such as [eclipse attacks][topic eclipse attacks].
+  When Bitcoin Core nodes receive an address message containing 10 or fewer
+  addresses, they forward it to 1 or 2 peers. This is the primary technique
+  used to self-advertise addresses, so sending to peers that would not relay
+  these addresses would effectively stop or "black hole" the trickle through
+  the network. Although this cannot be prevented in the malicious case, this
+  patch improves it for the honest cases, such as for block-relay-only
+  connections or light clients.
+
+  This update identifies whether or not an inbound connection is a
+  candidate for forwarding addresses based on whether it has sent an address
+  related message over the connection, such as `addr`, `addrv2`, or `getaddr`. This behavior
+  change could be problematic if there is software on the network that
+  relies on receiving address messages but never initiates an address-related
+  message. Therefore, the author took care to circulate this proposed change
+  before it was merged, including posting it to the
+  [mailing list][addrRelay improvements] and researching
+  [other open source clients][addr client research] to confirm compatibility.
+
+- [LND #5484][] allows it to store all data in a single external `etcd`
+  database. This improves high-availability deployments by making cluster
```

Comment:
'it' refers to LND, which hasn't been referred to before in the sentence, so this is a little bit awkward. Perhaps "LND #5484 allows the node to store all data in a single external `etcd` database." or "LND #5484 allows all of the node's data to be stored in a single external `etcd` database."